deps/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/deps/transport/http/HttpsRequestTest.java

```diff
@@ -26,6 +26,7 @@
 
 /** Unit tests for HttpRequest.
  * Coverage 66% method, 75% line */
+@SuppressWarnings("EmptyMethod")
```

Comment:
These suppressions are because we mock a function within the class and intentionally have that function do nothing